### DISCUSSION TITLE
MultilabelStratifiesFold

### DISCUSSION THREAD

**Discussion by kwakjeeyoon** [boostcampaitech2/object-detection-level2-cv-06#124] [ideas]
MultilabelStratifiesFold을 진행해 볼 생각입니다! 
class, number of boxes, median of object areas의 분포를 기준으로 train/val을 나누는 cross-validation 기법입니다!
아래 깃헙에서 제공되는 library가 있어 사용해 결과 내보겠습니다!

https://github.com/trent-b/iterative-stratification

**Comment by JiyouSeo** [comment]
지윤님 열일하시네요 ! 화이팅입니다 ~
